### DISCUSSION TITLE
Extension activation failed: "Contact Support. You are currently logged in as zhangzhr4."

### DISCUSSION THREAD

**Discussion by zhangzhr4** [community/community#116784] [Copilot Conversations]
### Select Topic Area

Question

### Body

![error2](https://github.com/community/community/assets/114235998/26a57378-4d97-4855-92e8-75a8f06d7ea3)
I have the student benefit to use copilot for free, but when I use it in vscode, it seems that the token is missing, and reconnect to github cannot solve the problem, may I fix this problem?
![error](https://github.com/community/community/assets/114235998/5ddace4d-6c09-4230-b1a9-69c5b8aad725)

**Comment by ayim** [comment]
@zhangzhr4 I recommend you contact GH support at https://support.github.com/contact/account on this issue.
There is nothing further that we can do to assist on an issue that explicitly says to Contact Support.
